solutions/cses-2079.mdx

```diff
@@ -126,5 +126,43 @@ public class Centroid {
 ```
 
 </JavaSection>
+	
+<PySection>
+
+```py
+	
+maxn = 200005
+g = [[] for i in range(maxn)] # graph
+s = [0 for i in range(maxn)] # size of subtree
+
+def dfs_size(cur: int, par: int) -> None:
```

Comment:
what does this do?

Reply:
unaddressed

Reply:
https://github.com/cpinitiative/usaco-guide/pull/3075/commits/30723be695212664dc78e397c527270309f424a0 

Please check the latest commit

Reply:
yeah? it doesn't put a comment saying what `dfs_size` does

Reply:
added it below line 138

Reply:
that...doesn't tell me what the function does
i'd add a doc comment saying "calculates all subtree sizes"

Reply:
updated it